test_pytest/test_c_contiguous.py

```diff
@@ -0,0 +1,35 @@
+import numpy as np
+
+
+def test_c_contiguous_array_input(request):
+    """Test whether cpp sides receives contiguous array"""
+
+    splinelist = request.getfixturevalue("spline_types_as_list")
+    splinedict = request.getfixturevalue("get_spline_dictionaries")
```

Comment:
I thought we were gonna get rid of those fixtures

Reply:
Yes, we will - however, it didn't work properly in first place, so I decided to move on in order to lose no time. Nevertheless, I'm going to change it later anyways.

Reply:
Solved